### PR TITLE
Save widgets' size along with their position

### PR DESCRIPTION
**Pull request by edmundoa**
In order to make widgets resizable, we need to store their size. I decided to do that along with the positions inside the dashboard, as it feels like it's something that belongs to the dashboard more than the widget itself.

I think I added the `height` and `width` properties on all relevant places, but it would be good to double-check it :)

These changes should play nicely with the current master branch of the web interface, in case you want to play with it.

### CODE REVIEW

File: graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java [hunk 1]
```diff
@@ -16,13 +16,12 @@
  */
 package org.graylog2.restclient.models.api.requests.dashboards;
 
-/**
- * @author Lennart Koopmann <lennart@torch.sh>
- */
 public class WidgetPositionRequest {
```
Comment (joschi) [inline graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java]: Would it make sense to consolidate this class with `org.graylog2.rest.resources.dashboards.requests.org.graylog2.rest.resources.dashboards.requests` and move it into graylog2-rest-models?

Comment (edmundoa) [inline graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java]: I guess you mean to consolidate it with `org.graylog2.rest.resources.dashboards.requests.WidgetPositions`. I will take care of that.

Comment (joschi) [inline graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java]: Yes, indeed I do. Copy & Paste fail. :tired_face: 

I have no idea what I'm doing

Comment (edmundoa) [inline graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java]: tumblr_midq2lnr3r1qgacvxo1_400

**Comment by edmundoa** [comment]
I refactored it as promised, I hope it looks a bit better now!

**Comment by joschi** [comment]
LGTM.

**Pull request merged**